clothing/views.py

```diff
@@ -38,6 +41,27 @@ def done(self, form_list, form_dict, **kwargs):
         for form in form_dict.get('1'):
             if 'type' in form.cleaned_data:
                 form.save()
+
+        email_addr = self.get_cleaned_data_for_step("0")['email']
+        person = Person.get_by_email_address_current(email_addr)
+        orders = '\n'.join(o.info() for o in Order.get_current(person=person))
```

Comment:
There should be a handling if there are no orders. Otherwise the email to the user looks a little strange. Something like:
```python
if orders == "":
    orders = _("Keine Bestellungen")
```